src/main/kotlin/org/stellar/kotlinwalletsdk/Wallet.kt

```diff
@@ -0,0 +1,5 @@
+package org.stellar.kotlinwalletsdk
```

Comment:
In common practice, package names do not use `kotlin`. 

for example: `org.stellar.walletsdk` or `org.stellar.wallet.sdk`

Reply:
If we later create Wallet SDK in Java, how would we name it? I added Kotlin there for that reason, but we can remove it.